resources/ext/ezbake.conf

```diff
@@ -21,7 +21,8 @@ ezbake: {
                          "mkdir -p /etc/puppet/manifests"],
                install: ["echo \\\"os-settings: {\\\"                         > $DESTDIR/$projconfdir/conf.d/os-settings.conf",
                          "echo \\\"    ruby-load-path: [$rubylibdir]\\\"      >> $DESTDIR/$projconfdir/conf.d/os-settings.conf",
-                         "echo \\\"}\\\"                                      >> $DESTDIR/$projconfdir/conf.d/os-settings.conf" ]
+                         "echo \\\"}\\\"                                      >> $DESTDIR/$projconfdir/conf.d/os-settings.conf" ],
+               pre-start-action: ["install --group={{user}} --owner={{user}} -d /var/run/puppet"]
```

Comment:
Yes, it does have to.  It's just sometimes not super clear when actions are carried out in EZBake.

In this case, pre-start-action gets mapped to `EZBake::Config[:redhat][:pre_start_action]` and the actions are carried out in the init scripts and unit files themselves, not in the packages.  Looks like this is what motivated [EZ-31](https://tickets.puppetlabs.com/browse/EZ-31) in the first place.

https://github.com/puppetlabs/ezbake/blob/e3f9ab7fafe6770733519ec6f435d5ef01d2eca8/resources/puppetlabs/lein-ezbake/template/foss/ext/redhat/init.erb#L60-L62